libs/packages/components/src/lib/video-player/video-player.component.ts

```diff
@@ -42,6 +42,8 @@ export class SdsVideoPlayerComponent implements AfterViewInit {
     }
 
     new InitPxVideo(this.config);
+    document.getElementsByClassName('px-video-container')[0].setAttribute("style", "width:"+this.VPConfiguration.width+";");
```

Comment:
I'd ideally try to avoid referencing document and instead go for injecting renderer or elementRef - some abstraction provided by Angular for DOM manipulation. Reference to document covers the entire page, so here, if a single page had 2 videos, then there'd be 2 video player components, and only the first video player would have its style configured.